### PR TITLE
Docs: uploaded and added docker section images

### PR DESCRIPTION
**Pull request by rkapoor10**
Files modified 
 - connectors -> DockerHub.md
 - secrets -> GithubPAT.md

### CODE REVIEW

File: docs/connectors/DockerHub.md [hunk 1]
```diff
@@ -3,18 +3,20 @@
 Follow these step to set up a Docker Connector
```
Comment (Ompragash) [inline docs/connectors/DockerHub.md:3]:
```suggestion
Follow these steps to set up a Docker Connector
```

**Review by Ompragash — CHANGES_REQUESTED** [review]
Great @rkapoor10! Added minor improvements, rest looks good

File: docs/connectors/DockerHub.md (hunk 1 above)
Comment (rkapoor10) [reply docs/connectors/DockerHub.md:3]: done

**Review by Ompragash — APPROVED** [review]
LGTM! Thanks @rkapoor10

**Pull request merged**